foyer/tests/implemented_opls_tests.txt

```diff
@@ -81,4 +81,17 @@ methanethiol
 t-butanol
```

Comment:
Lets go through and make sure we have the correct file names for some of this. For example, `t-butanol` is listed here but in the PR there is also a `tbutanol`